File: selector/src/main/java/com/luck/picture/lib/interfaces/OnRequestPermissionListener.java
package com.luck.picture.lib.interfaces;

/**
 * @author：luck
 * @date：2020/4/24 11:48 AM
 * @describe：OnRequestPermissionListener
 */
public interface OnRequestPermissionListener {
    /**
     * Permission request result
     *
     * @param permissionArray
     * @param isResult
     */
    void onCall(String[] permissionArray, boolean isResult);
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnRequestPermissionListener.kt
package com.luck.picture.lib.interfaces

/**
 * @author：luck
 * @date：2020/4/24 11:48 AM
 * @describe：OnRequestPermissionListener
 */
interface OnRequestPermissionListener {
    /**
     * Permission request result
     *
     * @param permission
     * @param isResult
     */
    fun onCall(permission: Array<String>, isResult: Boolean)
}